security/oc_acl.c

```diff
@@ -362,9 +365,89 @@ eval_access(oc_method_t method, uint16_t permission)
   return false;
 }
 
+static bool
+oc_sec_check_acl_on_get(const oc_resource_t *resource, bool is_otm)
+{
+  const char *uri = oc_string(resource->uri);
+  size_t uri_len = oc_string_len(resource->uri);
+
+  /* Retrieve requests to "/oic/res", "/oic/d" and "/oic/p" shall be granted. */
+  if (is_otm) {
+    if ((uri_len == 8 && memcmp(uri, "/oic/res", 8) == 0) ||
+        (uri_len == 6 && memcmp(uri, "/oic/d", 6) == 0) ||
+        (uri_len == 6 && memcmp(uri, "/oic/p", 6) == 0)) {
+      return true;
+    }
+
+#ifdef OC_HAS_FEATURE_PLGD_TIME
+    if (uri_len == OC_STRLEN(PLGD_TIME_URI) &&
+        memcmp(uri, PLGD_TIME_URI, OC_STRLEN(PLGD_TIME_URI)) == 0) {
+      return true;
+    }
```

Comment:
I read security papers yesterday about the device current time so it could be a public.

```
The current time itself may not be considered sensitive information because it is generally considered public information and is widely available through many sources such as clocks, watches, mobile devices, and other electronic devices.

The current time is not typically considered to be personally identifiable information (PII) or sensitive data because it does not reveal any personal or private information about an individual. It is a widely known and publicly available piece of information that is used for a variety of purposes, such as scheduling, timing events, or coordinating activities.
```

But only these properties could be accessible for anonymous(unsecure) connection:
```
time:  (current time)
rt:
if:
```